src/components/cylc/Tree.vue

```diff
@@ -35,13 +32,6 @@ export default {
       default: 0
     }
   },
-  watch: {
-    workflows: {
-      handler () {
-        this.clearCaches()
```

Comment:
This was clearing all cache. Even if we had expanded elements, or previously created elements, everything was destroyed, moving the cache to invalid states.